### DISCUSSION TITLE
reference/get_started/install_containers

### DISCUSSION THREAD

**Discussion by giscus** [devlive-community/datacap#413] [Q&A] [answered]
# reference/get_started/install_containers

DataCap is integrated software for data transformation, integration and visualization.

https://datacap.edurt.io/reference/get_started/install_containers.html

**Comment by iskaola** [comment]
docker的方式无法启动啊，需要修改数据库的配置，但是容器又启动失败进不去容器 ，死循环了这

**Comment by iskaola** [comment, marked as answer]
docker run -d -p 9909:9096  --name datacap -v /opt/datacap/configure/application.properties:/opt/app/datacap/configure/application.properties \ qianmoq/datacap


 挂载配置文件解决了。

**Reply by qianmoQ** [reply]
新版本的已经剔除了 h2 数据库所以无法使用，需要挂在外接配置

**Comment by Jonclex** [comment]
有预览的地方吗？gitee上的地址不能访问

**Reply by qianmoQ** [reply]
暂时关闭了预览的地址，目前可以使用 docker 方式进行本地部署，后续优化后会重新开放预览地址
